### PR TITLE
Artp 382/analytics tiles

### PR DESCRIPTION
**Pull request by snony**

### CODE REVIEW

File: src/client/src/ui/workspace/Workspace.tsx [hunk 1]
```diff
@@ -22,17 +24,19 @@ interface SpotTile {
 
 interface Props {
   spotTiles: SpotTile[]
+  tileView: TileViews
+  currencyView: CurrencyOptions
 }
 
-const Workspace: React.FC<Props> = ({ spotTiles = [] }) => (
+const Workspace: React.SFC<Props> = ({ spotTiles = [], tileView }) => (
```
Comment (spcfran) [inline src/client/src/ui/workspace/Workspace.tsx]: React.SFC is deprecated - recommendation is to use React.FC

File: src/client/src/shell/routes/SpotRoute.tsx [hunk 2]
```diff
@@ -13,10 +14,14 @@ const SpotTileStyle = styled.div`
...
 
-export default ({ match }: RouteComponentProps<{ symbol: string }>) => (
-  <RouteWrapper>
-    <SpotTileStyle>
-      <SpotTileContainer id={match.params.symbol} />
-    </SpotTileStyle>
-  </RouteWrapper>
-)
+export default ({ location, match }: RouteComponentProps<{ symbol: string }>) => {
+  const queryString = location.search.split('?')[1]
```
Comment (spcfran) [inline src/client/src/shell/routes/SpotRoute.tsx]: Here I would go with creating some utility method for parsing the query string that includes checks for null etc. There are tried and tested solutions out there you can copy or take inspiration from

File: src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { seriesData } from '../test-resources/spotTileProps'
+
+const AnalyticsTileChartWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+`
+
+class AnalyticsTileChart extends React.Component {
+  render() {
+    const lineProps = { strokeDasharray: '4 3', stroke: '#737987', strokeOpacity: 0.9, strokeWidth: 0.8 }
```
Comment (spcfran) [inline src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx]: Haven't tested the changes myself, but just as a heads up - please make sure all colors work well with both light and dark themes

File: src/client/src/shell/routes/SpotRoute.tsx (hunk 2 above)
Comment (snony) [reply src/client/src/shell/routes/SpotRoute.tsx]: Thanks

File: src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx (hunk 3 above)
Comment (snony) [reply src/client/src/ui/spotTile/components/analyticsTile/AnalyticsTileChart.tsx]: Thanks. The colour of the line is invariant to the theme as per Zeplin design guidelines. Furthermore, I have not yet been able to manipulate the lines attributes outisde a Rechart Component.

**Pull request merged**